### PR TITLE
Add origami icon click redirect to discover page.

### LINKED ISSUE #162: [UI]: Clicking on origami icon in the sider should take to home(Discover)
Currently, there is no click handler on sider origami icon. We should add a link to Origami home on the icon image.
image

### PR DESCRIPTION
**Pull request by fristonio**
- add an onclick handler to origami icon in app.js to take user to discover page.
- fix prettier issues with App.js
- Fixes #162

### CODE REVIEW

File: Origami/src/components/App.js [hunk 1]
```diff
@@ -152,7 +153,8 @@ class App extends React.Component {
   }
 
   render() {
-    /** 
+    console.log("In render");
```
Comment (AvaisP) [inline Origami/src/components/App.js]: Remove

**Comment by uttu357** [comment]
@fristonio Fix the conflicts.

**Comment by fristonio** [comment]
@uttu357 I have resolved the conflicts, can you have a look?

**Comment by uttu357** [comment]
Remove linting changes from this PR for it to be reviewed.
Make all the linting changes in #187.

**Comment by fristonio** [comment]
@uttu357 please review, I have rebased the branch with master(which have all lint errors resolved).

**Pull request merged**